Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -363,14 +377,20 @@ double score(const std::vector<size_t> &permutation,
 
     for (std::map<std::string, int>::const_iterator it = matchSet.begin();
          it != matchSet.end(); ++it) {
-      
+#ifdef DEBUG
+      std::cerr << " equiv: " << it->first << " " << it->second << " "
+                << permutation.size() << std::endl;
+#endif
+
       // if the rgroup is a hydrogens, only consider if the group is all
```

Comment:
might as well fix the typo here.  `Is a hydrogens` probably tells you the mental state I was in when I wrote this.